package.json

```diff
@@ -1,43 +1,44 @@
 {
-	"name": "jquery-migrate",
```

Comment:
Yeah no problem there, I am surprised that indentation survived for so long. :smile: